test-suit/starryos/normal/test-mt-execve/c/src/test_framework.h

```
@@ -0,0 +1,36 @@
+#pragma once
+
+#ifndef _GNU_SOURCE
+#define _GNU_SOURCE
+#endif
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <errno.h>
+
+static int __pass = 0;
```

Clang (14.0.6)

[warning] declaration uses identifier '__pass', which is a reserved identifier

(bugprone-reserved-identifier)